### DISCUSSION TITLE
After upgrade to 3.1.1, my postal is sending less number of emails. Worker > Thread config missing in new config support

### DISCUSSION THREAD

**Discussion by vikasmic** [postalserver/postal#2879] [Help with using Postal] [answered]
## Describe the bug
I have upgraded the postal server from 2.3.0/1 to 3.1.1

Post upgrade, my postal is just sending 7-8 emails / minute. 

In yaml, I am not able to find worker configuration

https://github.com/postalserver/postal/blob/main/doc/config/yaml.yml

  quantity: 8
  threads: 8

## To Reproduce

1. Go to 'https://docs.postalserver.io/getting-started/upgrade-to-v2'
2. Follow the Steps
3. See the sending results. 

## Expected behaviour

There was a thread configuration in the config file on older version, but its not there on new version and it seems causing a slow email sending issue to my server. 

## Screenshots

![image](https://github.com/postalserver/postal/assets/59131976/118a2efc-9b46-4534-8a49-72d594c38e3c)

## Environment details 

 - OS: Ubuntu 22.04
 - Browser Chrome

**Comment by willpower232** [comment]
In version 3 (and version 2) the easier way to scale is to add more workers via the `--scale` argument, I'm not sure off the top of my head if this is documented so not sure what @adamcooke would suggest as the best way for horizontal scaling nowadays.

**Comment by adamcooke** [comment, marked as answer]
7-8 emails per minute is extremely slow for a single worker. A single worker process can handle significantly more than that on a system that is configured correctly and not sending e-mails to SMTP servers on the moon.

However, the correct way to scale workers on v3 is to use a `docker-compose.override.yml` file in `/opt/postal/install`. The old worker configuration never worked well anyway.

```yaml
version: "3.9"
services:
  worker:
    deploy:
      replicas: 4
```

**Reply by gitbnitin** [reply]
Thanks @adamcooke @willpower232

**Reply by gitbnitin** [reply]
@willpower232 is there a way to set up a thread count as well?

**Reply by adamcooke** [reply]
No, the thread count on v3 is always 2 for now.

**Reply by gitbnitin** [reply]
Thanks @adamcooke
